_opcodes/op232-bgee.html

```diff
@@ -12,98 +12,175 @@
 pst: 0
 ---
 Casts the spell specified by the <code>resource</code> field on the target specified by the <code>Target</code> field, when the condition from the <code>Condition</code> field is true. The conditions are akin to various triggers.
-<br/>Known values for <code>Target</code> are:
+<br/><br/>
+
+Known values for <code>Target</code> are:
 <ul>
-	<li>0 &longrightarrow; Myself</li>
-	<li>1 &longrightarrow; LastHitter</li>
+	<li>0 &longrightarrow; <code><a href="../files/ids/bgee/object.htm#1">Myself</a></code></li>
+	<li>1 &longrightarrow; <code><a href="../files/ids/bgee/object.htm#LastHitter">LastHitter</a></code></li>
 	<li>
-		2 &longrightarrow; [ENEMY]
+		2 &longrightarrow; <code><a href="../files/ids/bgee/ea.htm#200">[EVILCUTOFF]</a></code>
 		<ul>
 			<li>Since object specifiers are not relative to the creature, this works as expected only from the Player's perspective.</li>
 		</ul>
 	</li>
-	<li>3 &longrightarrow; Nearest</li>
+	<li>3 &longrightarrow; <code><a href="../files/ids/bgee/object.htm#14">Nearest</a></code></li>
 </ul>
 
-<br/><br/>Known values for <code>Condition</code> are:
+Known values for <code>Condition</code> are:
 <ul>
-	<li>0 &longrightarrow; <a href="../scripting/triggers/bgeetriggers.htm#0x0020">HitBy([ANYONE])</a> – Checked every time</li>
+	<li>0 &longrightarrow; <code><a href="../scripting/triggers/bgeetriggers.htm#0x0020">HitBy([ANYONE])</a></code> &mdash; Checked every time</li>
 	<li>
-		1 &longrightarrow; <a href="../scripting/triggers/bgeetriggers.htm#0x401C">See([ENEMY])</a> – Checked once per round
+		1 &longrightarrow; <code><a href="../scripting/triggers/bgeetriggers.htm#0x401C">See([EVILCUTOFF])</a></code> &mdash; Checked once per round
```

Comment:
Are you sure these two ENEMY changes are correct? How would contingencies work for enemies then?

Reply:
Yes, they are correct (see [here](https://www.gibberlings3.net/forums/topic/34988-bg2ee-v2660-op232-corrections-special-flags/)).

So I think that instead of just `255|ENEMY`, any `EA` value belonging to the interval `[200–255]` is eligible...
And if the `op232` effect (Condition: `See([EVILCUTOFF])`) is attached to an `ENEMY`, then there must be another creature whose `EA` is any value in `[200–255]`...

And that's why it might not always work as expected.
It's not like `NearestEnemyOf()`, which returns the nearest creature of _opposite_ allegiance... It always refers to `EA` values `[200–255]`: so in case of `PC` (`EA=2`), then yes, you need a creature of opposite allegiance... but that's generally not true for NPCs.

Reply:
> > any EA value belonging to the interval [200–255] is eligible..
> 
> Well that's how these CUTOFF values work in general.

I'll ask Bubb for confirmation if necessary...